Backend/Interview.Backend/DateTimeJsonConverter.cs

```diff
@@ -0,0 +1,22 @@
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Interview.Backend;
+
+public class DateTimeJsonConverter : JsonConverter<DateTime>
+{
+    public const string DefaultDateTimeFormat = "yyyy-MM-ddTHH:mm:ss.fffZ";
+
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        var str = reader.GetString() ?? string.Empty;
+        return DateTime.Parse(str, null, DateTimeStyles.RoundtripKind);
```

Comment:
check this code, because I send "2024-07-03T15:45:36.416Z" but will saved "2024-07-03T18:45:36.416Z" Moscow time (the my current time)